csharp/src/Ice/Endpoint.cs

```diff
@@ -51,11 +45,8 @@ public abstract class Endpoint : IEquatable<Endpoint>
                 {
                     return option switch
                     {
-                        "compress" => HasCompressionFlag ? "true" : null,
                         "host" => Host.Length > 0 ? Host : null,
```

Comment:
compress and timeout should move to TcpEndpoint and UdpEndpoint, not disappear entirely.

Reply:
Fixed